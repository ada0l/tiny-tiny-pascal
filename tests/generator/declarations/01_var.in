var i: integer = 10; d: double = 2.3; s: string = 'hello, world';
begin writeln(i, ' ', d, ' ', s); end.