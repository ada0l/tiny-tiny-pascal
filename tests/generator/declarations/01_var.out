10 2.300000 hello, world
